TriangleCRM/TriangleAPI.php

```diff
@@ -1262,9 +1262,14 @@ private function ValidateCVVCount($paymentType,$cvv){
             $count = strlen($cvv);
             
             if($paymentType == 1){
-               $res = ($amexCount == $count)?true:false;
+    $res = ($amexCount == $count)?true:false; 
+       // fixing white space
+       
+       
             }else{
-                $res = ($other == $count)?true:false;
+                $res = ($other == $count)? (true ? false : true) :  (false ? false : true);`
```

Comment:
I don't like these hard coded booleans, you should abstract them into a lookup table
```
booleans = {
TRUE: true
FALSE: false
}
```